script/Ciel/Day 9/09_01_CIEL09_6.txt

```diff
@@ -861,44 +972,55 @@ But... something isn't right.
 [sha:1908a8ca7871caed8a04d06d17056fd40dd128c1]{
 -- Page 606, Offset 23689.
 -- 本来なら取るに足りない些細な間違いが、アイツを違う生き物に変えている―――
+// Unsure what exactly the 間違い is here. Biting her finger? Rising to Ciel's bait?
 Something that should've been a trifling blunder has instead transformed her into a different being.
 }
 [sha:bd88ed1203aeb5d5c4991af2029f19c486926747]{
 -- Page 607, Offset 23690.
 -- 「自由に、思うままに行動するというのですね。貴女が何の誓約もなく、思うままにロアを処罰すると。
 // random bullshit go (2)
-"To be free to act as you please, huh. If you were to punish Roa as you see fit, without forming any covenant, then―――
+// Mirroring Arcueid's 自由に from 23667.
+// Any lore behind 誓約 or is it just 'holding yourself to a standard'?
```

Comment:
She used to follow her 'instructions' even after the other TA were dead. Wake up, kill, etc.
Until now.